src/main/java/edu/ucsd/sbrg/escher/converter/Escher2SBML.java

```diff
@@ -41,7 +41,7 @@
  * This class takes an {@link EscherMap} as input and creates an
  * {@link SBMLDocument} based on the information provided.
  *
- * @author Andreas Dr&auml;ger
+ * @author Andreas Dr&auml;ger, Eike Pertuch
```

Comment:
It would be better to create a new line with a separate `@author` entry (for next time).